src/accumulation/accumulation_strategy.py

```diff
@@ -9,14 +9,16 @@ class AccumulationStrategy(ABC):
 
     @abstractmethod
     def on_merge(self,
-                 initial_point_cloud: np.ndarray,
-                 next_point_cloud: np.ndarray) -> np.ndarray:
+                 initial_point_cloud: np.ndarray[float],
```

Comment:
python3.8 does not support 'generic'-like typing. Unfortunately, we should use just _np.ndarray_

Reply:
removed